cluster/ci/config/roles.yaml

```diff
@@ -22,6 +22,12 @@ items:
   metadata:
     name: telemeter
 
+- kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    name: qe-jenkins
+    namespace: ocp
```

Comment:
It doesn't need to be in the ocp namespace.  Any service account token on the cluster can access these, so create a QE specific project and we'll assign it there.